examples/data-objects/focus-tracker/src/FocusTracker.ts

```diff
@@ -55,25 +78,38 @@ export class FocusTracker extends TypedEventEmitter<IFocusTrackerEvents> {
 
         this.audience.on("memberAdded", (clientId: string, member: IMember) => {
             this.emit("focusChanged");
+            this.emit("mousePositionChanged");
```

Comment:
I think these events are probably not necessary?  At the time the member is added, we won't yet know their focus or mouse position (because they will not have sent any signals yet) -- so at the time these events fire there won't be a change in the data.

Reply:
Yup, you're right!